packages/fdc3-context/generated/context/ContextTypes.ts

```diff
@@ -2139,1013 +2129,910 @@ export interface Valuation {
+        { json: "type", js: "type", typ: r("ContactListType") },
+    ], "any"),
+    "Context": o([
+        { json: "id", js: "id", typ: u(undefined, m("any")) },
+        { json: "name", js: "name", typ: u(undefined, "") },
+        { json: "type", js: "type", typ: "" },
+    ], "any"),
+    "Country": o([
+        { json: "id", js: "id", typ: r("CountryID") },
+        { json: "name", js: "name", typ: u(undefined, "") },
+        { json: "type", js: "type", typ: r("CountryType") },
+    ], "any"),
+    "CountryID": o([
+        { json: "COUNTRY_ISOALPHA2", js: "COUNTRY_ISOALPHA2", typ: u(undefined, "") },
+        { json: "COUNTRY_ISOALPHA3", js: "COUNTRY_ISOALPHA3", typ: u(undefined, "") },
+        { json: "ISOALPHA2", js: "ISOALPHA2", typ: u(undefined, "") },
+        { json: "ISOALPHA3", js: "ISOALPHA3", typ: u(undefined, "") },
+    ], "any"),
+    "Currency": o([
+        { json: "id", js: "id", typ: r("CurrencyID") },
+        { json: "name", js: "name", typ: u(undefined, "") },
+        { json: "type", js: "type", typ: r("CurrencyType") },
+    ], "any"),
+    "CurrencyID": o([
+        { json: "CURRENCY_ISOCODE", js: "CURRENCY_ISOCODE", typ: u(undefined, "") },
+    ], "any"),
+    "Email": o([
+        { json: "recipients", js: "recipients", typ: r("EmailRecipients") },
+        { json: "subject", js: "subject", typ: u(undefined, "") },
+        { json: "textBody", js: "textBody", typ: u(undefined, "") },
+        { json: "type", js: "type", typ: r("EmailType") },
+        { json: "id", js: "id", typ: u(undefined, m("any")) },
+        { json: "name", js: "name", typ: u(undefined, "") },
+    ], "any"),
+    "EmailRecipients": o([
+        { json: "id", js: "id", typ: u(undefined, r("ContactTIdentifiers")) },
+        { json: "name", js: "name", typ: u(undefined, "") },
+        { json: "type", js: "type", typ: r("EmailRecipientsType") },
+        { json: "contacts", js: "contacts", typ: u(undefined, a(r("ContactElement"))) },
+    ], "any"),
+    "ContactTIdentifiers": o([
+        { json: "email", js: "email", typ: u(undefined, "") },
+        { json: "FDS_ID", js: "FDS_ID", typ: u(undefined, "") },
+    ], "any"),
+    "FileAttachment": o([
+        { json: "data", js: "data", typ: r("FileAttachmentData") },
+        { json: "type", js: "type", typ: r("FileAttachmentType") },
+        { json: "id", js: "id", typ: u(undefined, m("any")) },
+        { json: "name", js: "name", typ: u(undefined, "") },
+    ], "any"),
+    "FileAttachmentData": o([
+        { json: "dataUri", js: "dataUri", typ: "" },
+        { json: "name", js: "name", typ: "" },
+    ], "any"),
+    "Instrument": o([
+        { json: "id", js: "id", typ: r("FluffyInstrumentIdentifiers") },
+        { json: "market", js: "market", typ: u(undefined, r("PurpleMarket")) },
+        { json: "name", js: "name", typ: u(undefined, "") },
+        { json: "type", js: "type", typ: r("PurpleInteractionType") },
+    ], "any"),
+    "FluffyInstrumentIdentifiers": o([
```

Comment:
Is "FluffyInstrument" a real thing?

Reply:
Its something that Quicktype creates when it has name conflicts. It creates type to represent properties of other types - their names don't really matter as they are just part of the top-level types structure. However, I do wish it didn't export them (assuming they can still be used as part of another exported tye) as they are not safe to use (a later change can trigger them getting generated with a different name).

Hmm, it might be worth post-processing the output with TSMorph and stopping them getting exported...